CHANGELOG.md

```diff
@@ -4,10 +4,7 @@
 - Rename `ValueOption` to `ToOption` in `IResult<T>` interface
 - Adds support for `@query` type
 - Adds support for user classes serialization
-- Adds support for new reference types in FaunaDB API 2.1
```

Comment:
Did we ever release this structures?

Reply:
No, I branched the version 1.3 and only release that version, 2.x.x was not released yet

Reply:
I mean, version 1.x doesn't have the new recursive references

Reply:
Ok then. I was just wondering if we need to do a separated release for this changes but, if they were not released yet we are good.